docroot/themes/humsci/humsci_basic/.gitignore

```diff
@@ -0,0 +1 @@
+dist/
```

Comment:
would it be better to keep this in the main .gitignore?

Reply:
Good call. It may be better to keep it all in the main `.gitignore` to avoid any confusion.

Reply:
hmm now do we want to keep the css artifact like the rest of the repo though?

Reply:
let's delete this ignore for now